libs/elements/layout/header/src/lib/components/header-menu/header-menu.component.scss

```diff
@@ -0,0 +1,25 @@
+.menu-items{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1200px;
+    margin-inline: auto;
+    z-index: 50000000000000;
+    background-color: white;
```

Comment:
For reuse of colors, use CSS variables specified in elements/theming/_variables.scss  for specifying colors and background colors